streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -293,17 +290,16 @@ public void onPartitionsRevoked(final Collection<TopicPartition> assignment) {
                     // suspend active tasks
                     taskManager.suspendTasksAndState();
                 } catch (final Throwable t) {
-                    log.error("{} Error caught during partition revocation, " +
-                            "will abort the current process and re-throw at the end of rebalance: {}", logPrefix, t.getMessage());
+                    log.error("Error caught during partition revocation, " +
+                            "will abort the current process and re-throw at the end of rebalance: {}", t.getMessage());
                     streamThread.setRebalanceException(t);
                 } finally {
                     streamThread.refreshMetadataState();
                     streamThread.clearStandbyRecords();
 
-                    log.info("{} partition revocation took {} ms.\n" +
+                    log.info("partition revocation took {} ms.\n" +
```

Comment:
same here as above

Reply:
`completed` is in lower case because before this we have applied a `logPrefix` using `LogContext`